Project_CA/FileChanges.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Project_CA
+{
+    public class FileChanges : Functionality
+    {
+        static float squery;
```

Comment:
Why do you need this variables here(not inside methods)? It's the best idea to have general variables for different methods.